server/game/gamesteps/actionwindow.js

```diff
@@ -22,9 +22,9 @@ class ActionWindow extends PlayerOrderPrompt {
 
     activePrompt() {
         return {
-            menuTitle: 'Any actions or reactions?',
+            menuTitle: 'Initiate an action, or pass',
```

Comment:
I like the latter for Reaction/Interrupt windows where multiple abilities could be listed. For actions windows the former has my preference, as the latter could possibly be construed as permission to initiate multiple actions in a row.